MediaBrowser.Model/Dlna/ContentFeatureBuilder.cs

```diff
@@ -151,10 +151,11 @@ public class ContentFeatureBuilder
                             DlnaFlags.InteractiveTransferMode |
                             DlnaFlags.DlnaV15;
 
-            // if (isDirectStream)
-            // {
-            //     flagValue = flagValue | DlnaFlags.ByteBasedSeek;
-            // }
+            if (isDirectStream)
+            {
+                flagValue |= DlnaFlags.ByteBasedSeek;
+            }
```

Comment:
Needs an extra linebreak
```suggestion
            }

```